packages/common/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../tsconfig-base.json",
+    "include": ["src/**/*"],
+    "compilerOptions": {
+        "outDir": "./lib/src"
```

Comment:
The other packages now use `lib` as the outDir. Are you keeping this to try to avoid breaking stuff?

Reply:
I'll change it to lib, like the others.